lighthouse-plugin-publisher-ads/computed/long-tasks.js

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const AdLanternMetric = require('../computed/ad-lantern-metric');
+const BaseNode = require('lighthouse/lighthouse-core/lib/dependency-graph/base-node.js');
+// @ts-ignore
+const ComputedMetric = require('lighthouse/lighthouse-core/computed/metrics/metric');
+// eslint-disable-next-line no-unused-vars
+const CpuNode = require('lighthouse/lighthouse-core/lib/dependency-graph/cpu-node.js');
+const {getAttributableUrl} = require('../utils/tasks');
+// @ts-ignore
+const LoadSimulator = require('lighthouse/lighthouse-core/computed/load-simulator');
+const MainThreadTasks = require('lighthouse/lighthouse-core/computed/main-thread-tasks');
+// @ts-ignore
+const makeComputedArtifact = require('lighthouse/lighthouse-core/computed/computed-artifact');
+// eslint-disable-next-line no-unused-vars
+const NetworkNode = require('lighthouse/lighthouse-core/lib/dependency-graph/network-node.js');
+const NetworkRecords = require('lighthouse/lighthouse-core/computed/network-records');
+const PageDependencyGraph = require('lighthouse/lighthouse-core/computed/page-dependency-graph');
+
+const LONG_TASK_THRESHOLD_MS = 50;
```

Comment:
Can you add a comment explaining why this is 50ms here, while it is 100ms in ad-blocking-tasks.js

Reply:
The difference is between what long tasks we collect vs. display. Added a comment in the audit